help/sources/tutorials/ui/create/customer-success/zendesk.md

```diff
@@ -0,0 +1,141 @@
+
+For more information on these credentials, see the Zendesk authentication documentation. 
+* [Where can I find my Zendesk subdomain](https://support.zendesk.com/hc/en-us/articles/4409381383578-Where-can-I-find-my-Zendesk-subdomain-)
+* [Generating a new API token](https://support.zendesk.com/hc/en-us/articles/4408889192858-Generating-a-new-API-token)
+
+Finally, create a Platform [schema](https://experienceleague.adobe.com/docs/experience-platform/xdm/schema/composition.html) required for the [!DNL Zendesk Search API]. Refer also to the [limits](#limits) section further below on this page.
+![Create Schema](../../../../images/tutorials/create/zendesk/schema.png?lang=en)
+
+## Connect your Zendesk account
+
+In the Platform UI, select **[!UICONTROL Sources]** from the left navigation bar to access the [!UICONTROL Sources] workspace. The [!UICONTROL Catalog] screen displays a variety of sources with which you can create an account.
+
+You can select the appropriate category from the catalog on the left-hand side of your screen. Alternatively, you can find the specific source you wish to work with using the search option.
+
+Under the *Customer Success* category, select *Zendesk*, and then select **[!UICONTROL Add data]**.
+
+![catalog](../../../../images/tutorials/create/zendesk/catalog.png)
+
+The **[!UICONTROL Connect Zendesk account]** page appears. On this page, you can either use new credentials or existing credentials.
+
+### Existing account
+
+To use an existing account, select the *Zendesk* account you want to create a new dataflow with, then select **[!UICONTROL Next]** to proceed.
+
+![existing](../../../../images/tutorials/create/zendesk/authentication-existing-account.png)
+
+### New account
+
+If you are creating a new account, select **[!UICONTROL New account]**, and then provide a name, an optional description, and your credentials. When finished, select **[!UICONTROL Connect to source]** and then allow some time for the new connection to establish.
+
+![new](../../../../images/tutorials/create/zendesk/authentication-new-account.png)
+
+## Next steps
+
+By following this tutorial, you have established a connection to your *Zendesk* account. You can now continue on to the next tutorial and [configure a dataflow to bring data into Platform](https://experienceleague.adobe.com/docs/experience-platform/sources/ui-tutorials/dataflow/crm.html).
+
+## Additional resources
+
+### Validation {#validation}
+
+To validate that you have correctly set up the source; and Zendesk profiles are being ingested, follow the steps below:
+
+1. In the Platform UI, select **[!UICONTROL Datasets]** from the left navigation bar to access the [!UICONTROL Datasets] workspace. The [!UICONTROL Dataset Activity] screen displays the details of executions.
+![Activity page](../../../../images/tutorials/create/zendesk/dataset-activity.png?lang=en)
+![Dataflow page](../../../../images/tutorials/create/zendesk/dataflow-monitoring.png?lang=en)
```

Comment:
Let's add a sentence between these two screenshots, describing the step that transitions between them. Something along the lines of: "Next, select the dataflow run ID of the dataflow that you want to view, to see specific details about that dataflow run."